[pt-br] Translation files from tsconfig-reference (#878)
microsoft/TypeScript-Website: packages/tsconfig-reference/copy/pt/categories/Strict_Type_Checking_Options_6173.md

---
display: "Verificações rígidas"
---

> Recomendamos utilizar [compiler option `strict`](#strict) para aceitar todas as melhorias possíveis a medida que são feitas.

Reviewer: ```suggestion
Recomendamos utilizar [a opção `strict`](#strict) para aceitar todas as melhorias possíveis a medida que são feitas.
```

TypeScript suporta um amplo espectro dos padrões do Javascript e padrões para permitir bastante flexibilidade em acomodar estes estilos.

> Freqüentemente, a segurança e escalabiliade em potencial do código base pode estar em conflito com alguma dessas técnicas.

Reviewer: ```suggestion
Frequentemente, a segurança e escalabiliade em potencial do código base pode estar em conflito com alguma dessas técnicas.
```

> Devido a variedade do Javascript suportado, a atualização para uma nova versão pode revelar dois tipos de erros:

Reviewer 2: ```suggestion
Devido a variedade do Javascript suportado, a atualização para uma nova versão do Typescript pode revelar dois tipos de erros:
```
Reduzir ambiguidade

> - Erros que já existem no seu código base, que o Typescript revelou pela linguagem melhorou seu entendimento do Javascript.

Reviewer 2: ```suggestion
- Erros que já existem no seu código base, que o Typescript passa a revelar, dado um melhor entendimento de Javascript.
```
Reduzir ambiguidade

> - Um novo conunto de erros que abordam um novo escopo de problema.

Reviewer 2: ```suggestion
- Um novo conjunto de erros que abordam um novo escopo de problema.
```

> O TypeScript irá adcionar um sinalizador de compilador para o último conjunto de erros, e por padrão estes não estão habilitados.

Reviewer 2: ```suggestion
O TypeScript irá adicionar um sinalizador de compilador para o último conjunto de erros, e por padrão estes não estão habilitados.
```
